language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -421,10 +525,155 @@ genModuleRef (pkgRef, modName) = case pkgRef of
 onHead :: (a -> a) -> [a] -> [a]
 onHead f = \case
     [] -> []
-    x:xs -> f x:xs
+    x : xs -> f x : xs
 
 onLast :: (a -> a) -> [a] -> [a]
 onLast f = \case
     [] -> []
     [l] -> [f l]
     x : xs -> x : onLast f xs
+
+writeTsConfig :: FilePath -> IO ()
+writeTsConfig dir = writeFileUTF8 (dir </> "tsconfig.json") $ unlines
+    [ "{"
+    , "  \"compilerOptions\": {"
+    , "    \"target\": \"es5\","
+    , "    \"lib\": ["
+    , "      \"dom\","
+    , "      \"es2015\""
+    , "     ],"
+    , "    \"strict\": true,"
+    , "    \"noUnusedLocals\": true,"
+    , "    \"noUnusedParameters\": false,"
+    , "    \"noImplicitReturns\": true,"
+    , "    \"noFallthroughCasesInSwitch\": true,"
+    , "    \"outDir\": \"lib\","
+    , "    \"module\": \"commonjs\","
+    , "    \"declaration\": true,"
+    , "    \"sourceMap\": true"
+    , "    },"
+    , "  \"include\": [\"src/**/*.ts\"],"
+    , "}"
+    ]
+
+writeEsLintConfig :: FilePath -> IO ()
+writeEsLintConfig dir = writeFileUTF8 (dir </> ".eslintrc.json") $ unlines
+  [ "{"
+  , "  \"parser\": \"@typescript-eslint/parser\","
+  , "  \"parserOptions\": {"
+  , "    \"project\": \"./tsconfig.json\""
+  , "  },"
+  , "  \"plugins\": ["
+  , "    \"@typescript-eslint\""
+  , "  ],"
+  , "  \"extends\": ["
+  , "    \"eslint:recommended\","
+  , "    \"plugin:@typescript-eslint/eslint-recommended\","
+  , "    \"plugin:@typescript-eslint/recommended\","
+  , "    \"plugin:@typescript-eslint/recommended-requiring-type-checking\""
+  , "  ],"
+  , "  \"rules\": {"
+  , "    \"@typescript-eslint/explicit-function-return-type\": \"off\","
+  , "    \"@typescript-eslint/no-inferrable-types\": \"off\""
+  , "  }"
+  , "}"
+  ]
+
+writePackageJson :: FilePath -> String -> String -> Scope -> [Dependency] -> IO ()
+writePackageJson packageDir damlTypesVersion packageVersion (Scope scope) depends =
+  writeFileUTF8 (packageDir </> "package.json") $ unlines
+  (["{"
+   , "  \"private\": true,"
+   , "  \"name\": \"" <> packageName <> "\","
+   , "  \"version\": \"" <> packageVersion <> "\","
+   , "  \"description\": \"Produced by daml2ts\","
+   , "  \"license\": \"Apache-2.0\","
```

Comment:
It's not clear to me that this is the right thing to do. For now, I would leave the `license` field out until we have a better idea.

Reply:
Ok.